<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Биномиальное распределение</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Гипергеометрическое распределение</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSpinBox" name="countSpinBox">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="prefix">
             <string>Введите количество столбцов: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="pSpinBox">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="prefix">
             <string>Введите p: </string>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="nSpinBox">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="prefix">
             <string>Введите n: </string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="dSpinBox">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="prefix">
             <string>Введите D: </string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="NSpinBox">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="prefix">
             <string>Введите N:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5"/>
        </item>
        <item>
         <widget class="QLabel" name="guide">
          <property name="text">
           <string> </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="formul">
          <property name="text">
           <string>formul</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <row>
           <property name="text">
            <string>xi</string>
           </property>
          </row>
          <row>
           <property name="text">
            <string>pi</string>
           </property>
          </row>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="resoltingButton">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Расчитать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="graphic">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Построить эмпирическую функцию распределения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="graphic2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Пострить полигон</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Mx">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Dx">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string> -</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
